axelrod/tests/unit/test_cooperator.py

```diff
@@ -43,13 +39,11 @@ class TestTrickyCooperator(TestPlayer):
     }
 
     def test_strategy(self):
-        """Starts by cooperating."""
+        # Starts by cooperating.
```

Comment:
Because the docstring doesn't fully describe the function and a lot of others were this way, so I standardized them. We can write better doc strings for these if you want but it's essentially always the same thing. The inline comments are better since they explain what each group of assertions is testing.